concepts/firstday/blood-gas-first-day-arterial.sql

```diff
@@ -0,0 +1,154 @@
+DROP TABLE IF EXISTS `physionet-data.mimic_derived.bloodgasfirstdayarterial`;
+CREATE TABLE `physionet-data.mimic_derived.bloodgasfirstdayarterial` AS
```

Comment:
you can delete this and use the first_day_bg_art table now - sorry to make you duplicate effort!

Reply:
Where is the `first_day_bg_art` table? I don't seem to see it under `physionet-data.mimic_derived`.

Reply:
it's now `bg` and you have to filter to arterial samples with `WHERE specimen_pred = 'ART.'` - I've added this to the sofa query now